Octokit.Tests/Reactive/Enterprise/ObservableEnterpriseAdminStatsClientTests.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using NSubstitute;
+using Octokit.Reactive;
+using Xunit;
+
+namespace Octokit.Tests
+{
+    public class ObservableEnterpriseAdminStatsClientTests
+    {
+        public class TheGetStatisticsIssuesMethod
+        {
+            [Fact]
+            public void CallsIntoClient()
+            {
+
+                var github = Substitute.For<IGitHubClient>();
+                var client = new ObservableEnterpriseAdminStatsClient(github);
+
+                client.GetStatisticsIssues();
+                github.Enterprise.AdminStats.Received(1).GetStatisticsIssues();
+            }
+        }
+
+        public class TheGetStatisticsHooksMethod
+        {
+            [Fact]
+            public void CallsIntoClient()
+            {
+                var github = Substitute.For<IGitHubClient>();
+                var client = new ObservableEnterpriseAdminStatsClient(github);
+
+                client.GetStatisticsHooks();
+                github.Enterprise.AdminStats.Received(1).GetStatisticsHooks();
+            }
+        }
+
+        public class TheGetStatisticsMilestonesMethod
+        {
+            [Fact]
+            public void CallsIntoClient()
+            {
+                var github = Substitute.For<IGitHubClient>();
+                var client = new ObservableEnterpriseAdminStatsClient(github);
+
+                client.GetStatisticsMilestones();
+                github.Enterprise.AdminStats.Received(1).GetStatisticsMilestones();
+            }
+        }
+
+        public class TheGetStatisticsOrgsMethod
+        {
+            [Fact]
+            public void CallsIntoClient()
+            {
+                var github = Substitute.For<IGitHubClient>();
+                var client = new ObservableEnterpriseAdminStatsClient(github);
+
+                client.GetStatisticsOrgs();
+                github.Enterprise.AdminStats.Received(1).GetStatisticsOrgs();
+            }
+        }
+
+        public class TheGetStatisticsCommentsMethod
+        {
+            [Fact]
+            public void CallsIntoClient()
+            {
+                {
```

Comment:
ah nice pickup...  Just pushed up that fix, hopefully the builds work :grimacing: